### PR TITLE
Add database import command and steps configured in harness.yml

### PR DESCRIPTION
**Pull request by g-foster2020**

### CODE REVIEW

File: src/akeneo/harness.yml [hunk 1]
```diff
@@ -30,6 +30,17 @@ attributes:
     user: akeneo
     pass: akeneo
     name: akeneo
+    import:
+      steps:
+        - passthru "echo \"Importing database file\""
+        - run "set -e -o pipefail; zcat \"$1\" | mysql -h \"$DB_HOST\" -u root -p\"$DB_ROOT_PASS\" \"$DB_NAME\""
```
Comment (kierenevans) [inline src/akeneo/harness.yml]: Missing the `pv` pipe via passthru - important for large databases.
https://github.com/inviqa/harness-base-php/blob/0.3.x/src/_base/docker/image/console/root/lib/task/assets/apply.sh#L28 has an example

File: src/akeneo/harness.yml [hunk 2]
```diff
@@ -30,6 +30,17 @@ attributes:
     user: akeneo
     pass: akeneo
     name: akeneo
+    import:
+      steps:
+        - passthru "echo \"Importing database file\""
```
Comment (kierenevans) [inline src/akeneo/harness.yml]: I would use single quotes if there's no variables inside the string

File: src/akeneo/harness.yml (hunk 1 above)
Comment (kierenevans) [reply src/akeneo/harness.yml]: Assumption of root user will conflict with https://github.com/inviqa/harness-base-php/pull/197

**Review by kierenevans — CHANGES_REQUESTED** [review]

File: src/_base/harness/config/commands.yml [hunk 3]
```diff
@@ -202,3 +202,9 @@ command('feature xdebug cli (on|off)'):
     run ws service php-fpm restart
     echo 'Done'
 
+command('db import <database_file>'):
+  env:
+    COMPOSE_PROJECT_NAME: = @('namespace')
+  exec: |
+    #!bash(workspace:/)|@
```
Comment (kierenevans) [inline src/_base/harness/config/commands.yml]: To use `={input.argument('database_file')}`, this needs to be:
```suggestion
    #!bash(workspace:/)|=
```

File: src/akeneo/harness.yml (hunk 1 above)
Comment (g-foster2020) [reply src/akeneo/harness.yml]: Yes some confusion there, I think there is more work to be done for #197 as there are still scripts in the harnesses using DB_ROOT_PASS, either that or I somehow don't have these changes.... I will check and remove DB_ROOT_PASS and convert anything else as per #197

Comment (g-foster2020) [reply src/akeneo/harness.yml]: Seems just my confusion.... Andy left in DB_ROOT_PASS I guess for backwards compatibility and misread this

File: src/_base/harness/config/commands.yml (hunk 3 above)
Comment (g-foster2020) [reply src/_base/harness/config/commands.yml]: The thing is.... it does work?

Comment (g-foster2020) [reply src/_base/harness/config/commands.yml]: So something wrong with the ws code expected result.  I will still change it as I can see the intention.

**Review by kierenevans — APPROVED** [review]

**Pull request merged**